### PR TITLE
Fix local wheel install

### PR DESCRIPTION
**Pull request by benjeffery**
Spotted by @grahamgower

### CODE REVIEW

File: .github/workflows/wheels.yml [hunk 1]
```diff
@@ -154,7 +154,7 @@ jobs:
         run: |
           python -VV
           # Install the local wheel
-          pip install tskit --only-binary tskit -f .
+          pip install pip install dist/tskit-*.whl
```
Comment (jeromekelleher) [inline .github/workflows/wheels.yml:157]: Looks like an extra "pip install"?

**Review by jeromekelleher — CHANGES_REQUESTED** [review]
Doesn't look right to me

**Comment by benjeffery** [comment]
Wow, I even read through the diff. Send help! Fixed.

**Review by jeromekelleher — APPROVED** [review]
LGTM

**Pull request merged**